site/source/docs/compiling/Deploying-Pages.rst

```diff
@@ -0,0 +1,162 @@
+.. _Deploying-Pages:
+
+===================================
+Deploying Emscripten Compiled Pages
+===================================
+
+Emscripten compiled output can either be run directly in a JS shell from command line, or hosted on a web page. When hosting asm.js and WebAssembly compiled pages as .html for browsers to execute, Emscripten provides a default `HTML shell file` that serves as a launcher to run the code, simplified to get started with development. However when getting ready to release and host the content on a web site, a number of extra features and customizations are likely in order to polish the visitor experience. This guide highlights things to pay attention to when deploying sites to a CDN.
+
```

Comment:
also, the last sentence sort of gives the impression that if you aren't using a CDN, then this doc isn't relevant, but is probably not the intention?